app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    copy_command = command_list[0]
+    file_in_name = command_list[1]
+    file_out_name = command_list[2]
```

Comment:
Instead of this use unzipping:

copy_command, file_in_name, file_out_name = command_list
Also you must validate that command_list have 3 parameters